src/ucp/core/ucp_context.c

```diff
@@ -314,6 +314,11 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "another thread, or incoming active messages, but consumes more resources.",
    ucs_offsetof(ucp_context_config_t, flush_worker_eps), UCS_CONFIG_TYPE_BOOL},
 
+  {"WORKER_STRONG_FENCE", "auto",
+   "Enable strong sync mode for ucp_worker_fence. By default it is used when\n"
+    "multiple lanes are used for RMA operations.",
```

Comment:
1. indent
2. Maybe it could be be enum "UCX_FENCE_MODE=[weak|strong|auto]" so we could add more options in the future?